.eslintrc.json

```diff
@@ -3,7 +3,10 @@
         "node": true,
         "es6": true
     },
-    "extends": ["airbnb-base"],
+    "extends": ["plugin:vue/essential"],
```

Comment:
this should be added to the lint config of the generated app, not here, here it should be only about the generator itself